www/tenfourfox/Portfile

```diff
@@ -25,17 +26,31 @@ if {${subport} eq "${name}"} {
     checksums           rmd160  e9a499d3acfe69c622eeafc7ea68ef957b7bdf3c \
                         sha256  b5aef01b6d031e4b582460955e3a3b95f7b3bfb994cdfa0171f0cd6279ca21ce \
                         size    325688845
+    set appname 	TenFourFox
 
 } elseif {${subport} eq "${name}-devel"} {
     conflicts           ${name}
     description         Clone of Firefox, compatible with macOS 10.4+. Tracks latest commit on GitHub.
     long_description    ${description}
 
-    github.setup        classilla tenfourfox 656875dfebe724c31896e3073780638e2c213f81
-    version             20230921-FPR32
-    checksums           rmd160  82d5229d46ab25ec9850754d97fd4f7ccad8c5e3 \
-                        sha256  a523c6a9985f1df83c167839b36f8c3ed6a67e133baf42859cf0d826e3d17bda \
-                        size    326149990
+    github.setup        classilla tenfourfox cdc8b5a989ce0b8ced48edbf8166d460c67a6a0d
+    version             20240505-FPR32
+    checksums           rmd160  3ad5aa53b5d127c8d46da03cb9e94e5573193db8 \
+                        sha256  2a9bf00d8b535cc78111f1f9ece6c185aeac3635b08dad09c9efb082356d3631 \
+                        size    326144707
+    set appname 	TenFourFox
+
+} elseif {${subport} eq "aquafox"} {
+    description         TenFourFox fork with extra optimizations.
+    long_description    ${description}
+
+    github.setup        BlackBirdLC Aquafox v1.0.0
```

Comment:
It should be:
```
github.setup        BlackBirdLC Aquafox 1.0.0 v
```
Otherwise the version will be prefixed with `v`, we do not want that.